featuretools/tests/demo_tests/test_demo_data.py

```diff
@@ -1,29 +1,42 @@
+try:
```

Comment:
If this is a python2 / 3 thing can we use the `is_python_2` check so it's easy to locate when we remove python2 support?

Reply:
Fixed
https://github.com/Featuretools/featuretools/pull/542/commits/8aef8890243b08a35cf59e4ab31d33405f2e7ee3